cms/static/js/views/paged_container.js

```diff
@@ -27,7 +26,33 @@ define(["jquery", "underscore", "js/views/utils/view_utils", "js/views/container
                     // of paginator, on the current page.
                     size: function() { return self.collection._size; },
                     // Toggles the functionality for showing and hiding child previews.
-                    showChildrenPreviews: true
+                    showChildrenPreviews: true,
+
+                    // PagingFooter expects to be able to control paging through the collection instead of the view,
```

Comment:
I'd prefer to see all the code refactored to go through the collection, so then these methods wouldn't be necessary. Did that seem too big a refactoring?